SU-SOE/stanford_soe_helper
# SOE-2061 added new mixin for text decoration colors and skips, passed through …

# READY FOR REVIEW

# Summary
- Added a mixin for text decorations
- Passed new mixin through all components

# Needed By (Date)
- 8/17 (end of sprint)

# Urgency
- No

# Steps to Test
- Pull this branch and verify that color underlines can be seen on Safari and iOS devices
- This branch is on /dev

# Affected Projects or Products
- SoE

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/SOE-2061

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
